improving_reliability_doesnt_always.md

```diff
@@ -0,0 +1,9 @@
+# Improving Reliability Doesn't Always
+
+Also known as [Lorin's Law](https://surfingcomplexity.blog/2017/06/24/a-conjecture-on-why-reliable-systems-fail/).
+
+Many major incidents are triggered by:
+* An attempt to resolve minor incidents through manual intervention
+* By a subsystem designed to improve reliability (gone haywire)
+
```

Comment:
Do you have any advice for avoiding this failure pattern?

Reply:
Added advice for avoiding the failure pattern but open to other suggestions.